frontend/src/App.tsx

```diff
@@ -1,24 +1,20 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
 
+import "./App.css";
+import Input from "./components/Input";
+import TodoList from "./components/TodoList";
+import { Todo } from "./models/todo.model";
+let todos: Todo[] = [];
+todos.push({ description: "Hello world", id: "244", isDone: true });
+todos.push({ description: "Hello Pakistan", id: "244", isDone: true });
+todos.push({ description: "Hello GB", id: "244", isDone: true });
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <div className="card">
+      <Input /> <button id="bt">Post</button>
```

Comment:
ids and classes should be name in a better way so its understandable `bt` is very random. You should follow bem naming convention https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/